hedy_translation.py

```diff
@@ -51,10 +52,11 @@ def translate_keywords(input_string, from_lang="en", to_lang="nl", level=1):
 
     punctuation_symbols = ['!', '?', '.']
 
+    keywordDict = keywords_to_dict(to_lang)
```

Comment:
deze var naam is niet zo "Pythonic". Dit is beter:
```suggestion
    keyword_dict = keywords_to_dict(to_lang)
```